scripts/changelog.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 The rkt Authors
+# Copyright 2020 Kinvolk
+#
+# Create a changelog.
+#
+# The env variable RANGE specifies the range of commits to be searched for the changelog.
+# If unset the latest tag until origin/master will be set.
+#
+# The env variable GITHUB_TOKEN can specify a GitHub personal access token.
+# Otherwise one could run into GitHub rate limits. Go to
+# https://github.com/settings/tokens to generate a token.
+
+set -e
+
+jq --version >/dev/null 2>&1 || {
+    echo "could not find jq (JSON command line processor), is it installed?"
+    exit 255
+}
+
+if [ -z "${RANGE}" ]; then
+    LATEST_TAG=$(git describe --abbrev=0)
+    RANGE="${LATEST_TAG}..origin/master"
+fi
+
+if [ ! -z "${GITHUB_TOKEN}" ]; then
+    GITHUB_AUTH="--header \"authorization: Bearer ${GITHUB_TOKEN}\""
+fi
+
+for pr in $(git log --pretty=%s --first-parent "${RANGE}" | egrep -o '#\w+' | tr -d '#'); do
```

Comment:
If I add `refs #100` to the commit message, to state, that the change is related to some issue, but not resolve it or anything, it will still show up?

Reply:
This is just so the releaser has something to start with when generating the changelog, they should decide what is in the changelog and what not in the end and heavily edit the output of this script.